components/SearchDropdown/index.tsx

```diff
@@ -0,0 +1,86 @@
+import styles from "../../styles/SearchDropdown/dropdown.module.css";
```

Comment:
Do we need this file to be in a folder?

IMO it can just be `/components/SearchDropdown.tsx`

Reply:
I was having the idea of keeping the style and js file in this folder. Since we are creating the styles in `styles` folder, I will remove this.